src/tribler/core/libtorrent/restapi/downloads_endpoint.py

```diff
@@ -1,14 +1,15 @@
 from __future__ import annotations
 
-from asyncio import CancelledError, wait_for
-from asyncio import TimeoutError as AsyncTimeoutError
+import mimetypes
+from asyncio import get_event_loop, shield, wait_for
 from binascii import hexlify, unhexlify
-from contextlib import suppress
 from pathlib import Path, PurePosixPath
 from typing import TYPE_CHECKING, Any, Optional, TypedDict, cast
 
 import libtorrent as lt
 from aiohttp import web
+from aiohttp.web_exceptions import HTTPPartialContent, HTTPRequestRangeNotSatisfiable
```

Comment:
Interesting: I never knew this existed. I guess we should start using this elsewhere, in Tribler and IPv8, too.